pkg/util/storage.go

```diff
@@ -0,0 +1,36 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2019 Datadog, Inc.
+
+package util
+
+import (
+	"github.com/DataDog/datadog-agent/pkg/config"
```

Comment:
We usually list standard import an empty line and then custom import:

```golang
￼	"io/ioutil"
￼	"path/filepath"
￼	"strings"

    "github.com/DataDog/datadog-agent/pkg/config"
￼	"github.com/DataDog/datadog-agent/pkg/util/log"
```

Reply:
OK I was hoping `fmt` handled that.